docs/source/_toctree.yml

```diff
@@ -16,7 +16,7 @@
   - local: installation
     title: Installation from source
   - local: supported_models
-    title: Supported Models and Hardware
+    title: Supported Models
```

Comment:
We also wanted to move this up in the section.

Reply:
WDYT?

Reply:
Yes! just after Quick tour is lovely! ty!